test/cmdline/helm_to_dp_test.go

```diff
@@ -132,5 +133,8 @@ func (s *TestSuite) TestHelmToDpBadURL() {
 
 	_, stderr, err := s.runHelmToDp(badHelmChart, "-o", tempDir)
 	s.Error(err, "Expected no error when running catalog-schema on a good package")
-	s.Contains(stderr, "failed to resolve", "Expected error failing to resolve the vad url")
+	s.True(
+		strings.Contains(stderr, "failed to resolve") || strings.Contains(stderr, "failed to verify certificate"),
+		"Expected error containing 'failed to resolve' or 'failed to verify certificate' in stderr",
```

Comment:
If we can avoid relaying on error message it would be better.

Reply:
I still do want to ensure that the REST API is returning appropriate error text. I'll think about what to do about it tomorrow. It's noteworthy that some error is occurring that was not what was expected. For tonight we'll remove the checks and get the component tests unblocked.